src/pywifes/wifes_wsol.py

```diff
@@ -890,14 +904,10 @@ def xcorr_shift_grid(slitlet_data, wave_guess, ref_interp):
     # pylab.plot(y_samp[good_inds], all_shifts[good_inds], 'bo')
     # pylab.plot(fval_y, shift_fvals, 'g')
     # pylab.show()

```

Comment:
Same for this commented piece of code.

Reply:
I think I can reuse this piece of code for an extra plot for checking wavelength solution. I'll leave it for now.